circular-buffer/circular_buffer_tests.erl

```diff
@@ -0,0 +1,54 @@
+-module( circular_buffer_tests ).
```

Comment:
This is the problem, the test runner expects this to be `-module(circular_buffer_tests).`